src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -803,6 +803,84 @@ public function testMappingDuplicateKeyFlow()
         $this->assertSame($expected, Yaml::parse($input));
     }
 
+    /**
+     * @dataProvider getParseExceptionOnDuplicateData
```

Comment:
missing note saying that this should turned to an `@expectedException` assertion on 4.0, isn't it?